pkg/backup/abs_backend_test.go

```diff
@@ -0,0 +1,193 @@
+// Copyright 2017 The etcd-operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backup
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/storage"
+	"github.com/coreos/etcd-operator/pkg/backup/abs"
+)
+
+const integrationTestEnvVar = "RUN_INTEGRATION_TEST"
+
+var (
+	accountName    = storage.StorageEmulatorAccountName
+	accountKey     = storage.StorageEmulatorAccountKey
+	DefaultBaseURL = "http://127.0.0.1:10000"
+	container      = "testcontainer"
```

Comment:
I am going to make the Azuite test container be shared among all test builds; Therefore, we need to randomized `container` to avoid test collision.